test/unit_test/passes/inc/test_inc_quantization.py

```diff
@@ -15,62 +14,57 @@
 from olive.passes.olive_pass import create_pass_from_dict
 from olive.passes.onnx.conversion import OnnxConversion
 from olive.passes.onnx.inc_quantization import IncDynamicQuantization, IncQuantization, IncStaticQuantization
-from olive.systems.local import LocalSystem
 
 
 @pytest.mark.skipif(
     platform.system() == "Windows", reason="Skip test on Windows. neural-compressor import is hanging on Windows."
 )
-def test_inc_quantization():
-    with tempfile.TemporaryDirectory() as tempdir:
-        # setup
-        ov_model = get_onnx_model(tempdir)
-        local_system = LocalSystem()
-        data_dir = Path(tempdir) / "data"
-        data_dir.mkdir(exist_ok=True)
-        config = {"data_dir": data_dir, "dataloader_func": create_dataloader}
-        output_folder = str(Path(tempdir) / "quantized")
-
-        # create IncQuantization pass
-        p = create_pass_from_dict(IncQuantization, config, disable_search=True)
-        # execute
-        quantized_model = local_system.run_pass(p, ov_model, None, output_folder)
-        # assert
-        assert quantized_model.model_path.endswith(".onnx")
-        assert Path(quantized_model.model_path).exists()
-        assert Path(quantized_model.model_path).is_file()
-        assert "QLinearConv" in [node.op_type for node in quantized_model.load_model().graph.node]
-
-        # clean
-        del p
-        # create IncDynamicQuantization pass
-        p = create_pass_from_dict(IncDynamicQuantization, config, disable_search=True)
-        # execute
-        quantized_model = local_system.run_pass(p, ov_model, None, output_folder)
-        # assert
-        assert quantized_model.model_path.endswith(".onnx")
-        assert Path(quantized_model.model_path).exists()
-        assert Path(quantized_model.model_path).is_file()
-        assert "DynamicQuantizeLinear" in [node.op_type for node in quantized_model.load_model().graph.node]
-
-        # clean
-        del p
-        # create IncStaticQuantization pass
-        p = create_pass_from_dict(IncStaticQuantization, config, disable_search=True)
-        # execute
-        quantized_model = local_system.run_pass(p, ov_model, None, output_folder)
-        # assert
-        assert quantized_model.model_path.endswith(".onnx")
-        assert Path(quantized_model.model_path).exists()
-        assert Path(quantized_model.model_path).is_file()
-        assert "QLinearConv" in [node.op_type for node in quantized_model.load_model().graph.node]
-
-
-def get_onnx_model(tempdir):
-    local_system = LocalSystem()
+def test_inc_quantization(tmpdir):

```

Comment:
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmpdir-and-tmpdir-factory-fixtures
Seems the `tmpdir` return legacy [py.path.local](https://py.readthedocs.io/en/latest/path.html) objects rather than standard [pathlib.Path](https://docs.python.org/3/library/pathlib.html#pathlib.Path) objects.

The documents suggest to use `tmp_path`

Reply:
Thanks! Follow up PR to use this https://github.com/microsoft/Olive/pull/550